### PR TITLE
Add video entity.md

### PR DESCRIPTION
**Pull request by medhaanand**

### CODE REVIEW

File: docs/bot-builder/basic/entities.md [hunk 1]
```diff
@@ -273,13 +273,29 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving a valid PANCARD image then the **entity reprompt message** would be asking the user for a valid image.
+
+**Video** attachment type is only for Whatsapp bots as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:292]: Can we highlight the Whatsapp piece? Get's lost while reading it.

Where do we mention the PII information details? Where the uploaded video will be deleted from our end in 3 days?
@RandiveSushant @medhaanand

File: docs/bot-builder/basic/entities.md [hunk 2]
```diff
@@ -273,13 +273,29 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+- Video
+
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving a valid PANCARD image then the **entity reprompt message** would be asking the user for a valid image.
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:290]: That is a confusing statement. We don't validate the images that are uploaded by the user.

If user upload AADHAR card image when asked for PANCARD, we won't be able to figure that out(except for human assistance)

File: docs/bot-builder/basic/entities.md [hunk 3]
```diff
@@ -273,13 +273,29 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
 
-![Attachment Entity](assets/AttachmentEntity.png)
+![Attachment Entity](assets/Video1.png)
 
-You would need to create a Button HSL for letting the user upload the attachment.
+There are these three types of attachments allowed -
+
+- Image
+- Document (pdf, docx, xls, csv)
+- Video
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:282]: Can we mention it here that this is only supported for Whatsapp?

File: docs/bot-builder/basic/entities.md [hunk 4]
```diff
@@ -273,13 +273,29 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving a valid PANCARD image then the **entity reprompt message** would be asking the user for a valid image.
+
+**Video** attachment type is only for Whatsapp bots as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -
+
+"Please wait while we process your video message."
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:294]: Is this needed to be specified explicitly? Adds confusion saying will this message be returned instead of Bot says?

File: docs/bot-builder/basic/entities.md [hunk 5]
```diff
@@ -273,13 +273,29 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+
+For video entity to be captured on a Whatsapp bot, we need to enable the settings from Business Manager as shown below -
+
+![Attachment Entity](assets/Video3.png)
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:298]: This looks good 👍

File: docs/bot-builder/basic/entities.md (hunk 4 above)
Comment (medhaanand) [reply docs/bot-builder/basic/entities.md:294]: Yes this needs to be specified here as or else it will increase confusion if not present. People might not know where is this message coming if they have not written it there

Comment (medhaanand) [reply docs/bot-builder/basic/entities.md:294]: changed the content to explain it better

File: docs/bot-builder/basic/entities.md (hunk 3 above)
Comment (medhaanand) [reply docs/bot-builder/basic/entities.md:282]: I have written that below, we can add it here too

Comment (medhaanand) [inline docs/bot-builder/basic/entities.md:282]:
```suggestion
- Video (Only for Whatsapp)
```

File: docs/bot-builder/basic/entities.md (hunk 2 above)
Comment (medhaanand) [reply docs/bot-builder/basic/entities.md:290]: check now

File: docs/bot-builder/basic/entities.md [hunk 6]
```diff
@@ -273,13 +273,29 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+- Video (Only for Whatsapp)
+
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving a valid PANCARD image then the **entity reprompt message** would be asking the user for a valid image.
```
Comment (medhaanand) [inline docs/bot-builder/basic/entities.md:290]:
```suggestion
In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.
```

File: docs/bot-builder/basic/entities.md [hunk 7]
```diff
@@ -273,13 +273,29 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.
+
+**Video** attachment type is only for Whatsapp bots as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -
```
Comment (medhaanand) [inline docs/bot-builder/basic/entities.md:292]:
```suggestion
**Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -
```

File: docs/bot-builder/basic/entities.md [hunk 8]
```diff
@@ -273,13 +273,31 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.
+
+**Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message -
```
Comment (medhaanand) [inline docs/bot-builder/basic/entities.md:292]:
```suggestion
**Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -
```

File: docs/bot-builder/basic/entities.md [hunk 9]
```diff
@@ -273,13 +273,31 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.
+
+**Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:292]: `we will be able to upload videos there` -> `the user will be able to upload videos there` maybe?

File: docs/bot-builder/basic/entities.md [hunk 10]
```diff
@@ -273,13 +273,31 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+- Video (Only for Whatsapp)
+
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not giving an image then the **entity reprompt message** would be asking the user for a valid input.
```
Comment (tejas-haptik) [inline docs/bot-builder/basic/entities.md:290]: `if the user is not giving an image` -> `if the user is not uploading an image file` maybe?

Comment (tejas-haptik) [reply docs/bot-builder/basic/entities.md:290]: `user for a valid input` maybe rephrase this.

Comment (tejas-haptik) [reply docs/bot-builder/basic/entities.md:290]: I understand these are not your changes, so you can skip them for now if it's too much to change.

Comment (medhaanand) [inline docs/bot-builder/basic/entities.md:290]:
```suggestion
In the above example, if the user is not uploading an image file then the **entity reprompt message** would ask the user for a valid input.
```

File: docs/bot-builder/basic/entities.md [hunk 11]
```diff
@@ -273,13 +273,31 @@ You can refer https://regex101.com/ to learn more about regular expression.
...
+![Attachment Entity](assets/Video2.png)
+
+You would need to create a Button HSL for letting the user upload the image and document. 
 
 ![Attachment Example on Bot](assets/AttachmentHSL.png)
 
-In the above example, if the user is not giving a valid PANCARD image then the entity reprompt message would be asking the user for a valid image.
+In the above example, if the user is not uploading an image file then the **entity reprompt message** would ask the user for a valid input.
+
+**Video** attachment type is only for **Whatsapp bots** as we will be able to upload videos there. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -
```
Comment (medhaanand) [inline docs/bot-builder/basic/entities.md:292]:
```suggestion
**Video** attachment type is only for **Whatsapp bots** as the user will be able to upload videos only on Whatsapp. Once you successfully upload the video, you will get an automated message along with your predefined Bot Says messages -
```

**Review by tejas-haptik — COMMENTED** [review]
Looks good 👍

**Review by tejas-haptik — APPROVED** [review]

**Review by RandiveSushant — APPROVED** [review]
